app/controllers/orgs/rosters_controller.rb

```diff
@@ -146,6 +174,61 @@ def download_roster
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/AbcSize
 
+    def select_google_classroom
+      @google_classroom_courses = fetch_all_google_classrooms
+    end
+
+    def search_google_classroom
+      courses_found = fetch_all_google_classrooms.select do |course|
+        course.where("name ILIKE ?", "#{params[:query]}#")
```

Comment:
Ah sorry, I think I may have left bad feedback here, I thought this returned an AR relation. It looks like this fires a SQL query per course which isn't great for performance. If this is an array and the data is already loaded, lets just do the filtering in memory here. Something like `course.name.downcase.include?(params[:query].downcase)`